### PR TITLE
Feature/pla 3701 list tables

### PR DESCRIPTION
**Pull request by dleblanc**
# Citrine Python PR

Tiny change to:
 - use the right collection key when listing tables
 - include the version number in the Table's __str__

### PR Type:
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

### Adherence to team decisions
- [ ] I have added tests for 100% coverage
- [ ] I have written Numpy-style docstrings for every method and class.
- [ ] I have communicated the downstream consequences of the PR to others.
- [ ] I have bumped the version in setup.py

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -57,7 +57,7 @@ def run(self):
 
 
 setup(name='citrine',
-      version='0.9.1',
+      version='0.9.2',
```
Comment (dekatzenel) [inline setup.py:60]: Version is out of date

Comment (dleblanc) [reply setup.py]: I must have missed something, seems like a large bump to go to .14?

Comment (dekatzenel) [reply setup.py]: You missed us starting to enforce in an automated way that all commits bump the version, plus a few separate commits that added new methods

Comment (dleblanc) [reply setup.py]: Ok, cool. I did actually bump the version initially too, but that must have gotten stale.

**Pull request merged**